style/style_script/create_styles.py

```diff
@@ -0,0 +1,182 @@
+# !/usr/bin/python2.7
```

Comment:
Probably worth making this new script compatible with 3, if possible:
https://python-future.org/compatible_idioms.html